frontend/src/component/insights/Insights.tsx

```diff
@@ -59,7 +59,12 @@ export const Insights: VFC = () => {
                             selectedProjects={projects}
                             onChange={setProjects}
                             dataTestId={'DASHBOARD_PROJECT_SELECT'}
-                            sx={{ flex: 1, maxWidth: '360px', width: '100%' }}
+                            sx={{
+                                flex: 1,
+                                maxWidth: '360px',
+                                width: '100%',
+                                zIndex: 100000,
```

Comment:
Maybe we can avoid fixed numbers here? https://mui.com/material-ui/customization/z-index/
https://github.com/Unleash/unleash/blob/main/frontend/src/themes/theme.ts#L73-L75

Reply:
Sure, this was there.  Feel free to adapt in a different PR.  The point of this PR is to fill the chart data with 0s when no data for that datapoint

Reply:
Removed the zIndex. It's not needed